kytos/core/user.py

```diff
@@ -62,12 +63,18 @@ def projection() -> dict:
             "username": 1,
             "email": 1,
             'password': 1,
+            'salt': 1,
             'state': 1,
             'inserted_at': 1,
             'updated_at': 1,
             'deleted_at': 1
         }
 
+    @staticmethod
+    def hashing(password, salt) -> str:
+        return hashlib.scrypt(password=password, salt=salt, n=8192,
+                              r=8, p=1).hex()
+
```

Comment:
@Alopalao let's also cover this requirement that was mentioned:

> -  JWT token expiration should be valid for at least 1 hour to minimize the number of login refresh

https://github.com/kytos-ng/kytos/blob/master/kytos/core/config.py#L148
https://github.com/kytos-ng/kytos/blob/master/kytos/templates/kytos.conf.template#L86

If you could first double check that with 180 it's working as intended and then please let's increase it to 3600 to avoid that callers need to be refreshing too frequently

Reply:
Yes it is expiring correctly.
![Screenshot from 2022-12-07 09-22-40](https://user-images.githubusercontent.com/55767214/206204964-d2e46d56-9ab1-4999-8d6c-cb85aaa2dde5.png)

Reply:
Awesome.